docs/basics/sso/setting-up-sso.md

```diff
@@ -15,6 +15,12 @@ Sauce Labs supports Identity Provider (IdP)-initiated and Service Provider (SP)-
 - An enterprise license for Sauce Labs
 - Organization admin access (see [User Roles](/basics/acct-team-mgmt/managing-user-info) for more information)
 
+## Special Cases
+
+If your SSO setup falls into one of these special cases, refer to these setup steps:
```

Comment:
```suggestion
```

Reply:
Do you want me to remove this line completely?

Reply:
Yes, we can add a intro sentence later when you add more cases.